### PR TITLE
Improvement: refactor step

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

**Review by jRimbault — DISMISSED** [review]

File: cypress/support/step_definitions/html.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Then } from '@badeball/cypress-cucumber-preprocessor';
+import nunjucks from 'nunjucks';
+
+Then('I expect {string} exists', (selector) => {
+  cy.get(selector).should('be.visible');
+});
+
+Then('I expect {string} is {string}', (selector, __value) => {
```
Comment (k-y0u) [inline cypress/support/step_definitions/html.js:8]: __value -> expectedValue (like previous commit)

File: cypress/support/step_definitions/context.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+});
+
+Then('I extract {string} from url in field {string} of context', (pattern, key) => {
+  cy.url().then((url) => {
+    const projectName = new RegExp(pattern).exec(url)[0];
+    cy.context[key] = projectName;
+  });
+});
+
+Then('I expect context field {string} is {string}', (key, value) => {
```
Comment (k-y0u) [inline cypress/support/step_definitions/context.js:14]: value -> expectedValue (like previous commit)

File: cypress/support/step_definitions/context.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  expect(cy.context[key]).to.eq(value);
+});
+
+Then('I set localstorage field {string} to context field {string}', (localStorageField, contextField) => {
+  cy.context[contextField] = localStorage.getItem(localStorageField);
+});
+
+Then('I set localstorage field {string} to context field {string} as {string}', (localStorageField, contextField, type) => {
+  if (type === 'json') {
```
Comment (k-y0u) [inline cypress/support/step_definitions/context.js:23]: no else part ?

File: README.md [hunk 4]
```diff
@@ -59,6 +59,7 @@ leto-modelizer
 ├ cypress              ⇨ Contains all the cypress related files
 │ ├ e2e                ⇨ Contains all the e2e tests
 │ ├ videos             ⇨ Contains all videos of each run test (only locally)
+│ └ support            ⇨ Contains all step definitions for e2e tests
```
Comment (k-y0u) [inline README.md:62]: │ └ support            ⇨ Contains all step definitions for e2e tests

TO

│ └ support            ⇨ Contains all supports files for e2e tests
│ │└ step_definitions            ⇨ Contains all step definitions for e2e tests

(something like this)

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -59,6 +59,8 @@ leto-modelizer
 ├ cypress              ⇨ Contains all the cypress related files
 │ ├ e2e                ⇨ Contains all the e2e tests
 │ ├ videos             ⇨ Contains all videos of each run test (only locally)
+│ └ support            ⇨ Contains all supports files for e2e tests
```
Comment (k-y0u) [inline README.md:62]: supports files ->  support files

**Review by k-y0u — APPROVED** [review]

**Review by jRimbault — APPROVED** [review]

**Pull request merged**